registry.json

```diff
@@ -3558,7 +3558,21 @@
     "telegram": "HippoOperator",
     "operatorName": "Jerome",
     "github": "HIPPO-Pool"
-  },   
+  },  
+  {
+    "ticker": "HELLO",
+    "poolId": "f9317946c36f6ea5625732382ba4de05ddf4a3fe5952caa838cee051",
+    "social": {
+      "website": "https://adattitude.github.io",
+      "twitter": "TICKER_HELLO",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "",
```

Comment:
Please can you add a TG handle? This must be the handle of a user and not a group or channel.